src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -1426,14 +1411,24 @@ void Graph::GetPerfData(std::vector<ov::ProfilingInfo>& perfMap) const {
     }
 }
 
+void Graph::CreateEdge(const NodePtr& parent,
+                       const NodePtr& child,
+                       int parentPort,
+                       int childPort) {
+    auto edge = std::make_shared<Edge>(parent, child, parentPort, childPort);
```

Comment:
Sounds like it worth to add some validation for port values (at least debug only):
1. Non negative value
2. Child doesn't have parent edge at this port

Reply:
Done